ci/docker/runtime_functions.sh

```diff
@@ -783,8 +783,26 @@ build_docs() {
 
 #Runs Apache RAT Check on MXNet Source for License Headers
 nightly_test_rat_check() {
-    set -ex
-    ./tests/nightly/apache_rat_license_check/license_check.sh
+    set -e
+    pushd .
+    
+    cd /work/deps/trunk/apache-rat/target
+
+    # Use shell number 5 to duplicate the log output. It get sprinted and stored in $OUTPUT at the same time https://stackoverflow.com/a/12451419
+    exec 5>&1
+    OUTPUT=$(java -jar apache-rat-0.13-SNAPSHOT.jar -E /work/mxnet/tests/nightly/apache_rat_license_check/rat-excludes -d /work/mxnet|tee >(cat - >&5))
```

Comment:
This may be unrelated but what is file descriptor 5 here. I tried on my machine and it gives bad file descriptor. And when I echo $OUTPUT it says : `Ignored 1 lines in your exclusion files as comments or empty lines.` Maybe its just my env, but it is this rat check verified ?

Reply:
It's basically just an empty shell number. 1 is stdout, 2 is stderr. I think the other ones are unused. This is basically a hack to mirror the output and store it in a variable https://stackoverflow.com/a/12451419f

Exactly. $OUTPUT will contain the output of the RAT check. If there are no problems, it will output what you have seen. If there are any problems, it will contain details and always print the string which is stored in ERROR_MESSAGE. We determine whether the run was successful by checking if the output contains that error message.

Reply:
I have unknown licenses in my repo and it contains the ERROR_MESSAGE. But if it is verified on your end I am okay with this.